ui/src/shared/components/MultiGrid/MultiGrid.tsx

```diff
@@ -23,15 +26,12 @@ interface Props {
   scrollTop?: number
   scrollLeft?: number
   rowCount?: number
-  rowHeight?: (arg: {index: number}) => {} | number
-  columnWidth?: (arg: object) => {} | number
+  rowHeight?: number | HeightWidthFunction
```

Comment:
Can `rowHeight` and `columnWidth` both be of type `HeightWidthFunction`?

Reply:
I feel like it's more descriptive here to indicated that it could be a regular number or a function that returns a number. ?